server/accounts.go

```diff
@@ -2889,6 +2885,21 @@ func (ur *URLAccResolver) Fetch(name string) (string, error) {
 	return string(body), nil
 }
 
+// Store will store or update the jwt claim of account `name`
+// to the base url, appending the account name onto the end.
+func (ur *URLAccResolver) Store(name, jwt string) error {
+	url := ur.url + name
+	resp, err := http.Post(url, "application/jwt", strings.NewReader(jwt))
+	if err != nil {
+		return err
+	} else if resp.StatusCode != http.StatusOK {
```

Comment:
After the first `if err != nil` you should first do a `defer resp.Body.Close()`. Then have this `if resp.StatusCode != ..` test.

Reply:
done